inst/settings.json

```diff
@@ -22,5 +22,13 @@
         "pos_y_ref": "pos_y_ref",
         "log10_ratio": "log10_ratio_conc",
         "log2_CI": "log2_CI"
-    }
-}
\ No newline at end of file
+    },
+    "assay_dt_req_uniq_col_ids": [
+        "drug_name",
+        "drug_name2",
```

Comment:
"gnumber" and "gnumber2" are not on this list?

Reply:
Yes. They've never been.

Reply:
✔️